prow/github/client.go

```diff
@@ -2316,3 +2316,75 @@ func (s *reloadingTokenSource) Token() (*oauth2.Token, error) {
 		AccessToken: string(s.getToken()),
 	}, nil
 }
+
+// GetOrgProjects returns the list of projects in this org.
+func (c *Client) GetOrgProjects(org string) ([]Project, error) {
+	c.log("GetOrgProjects", org)
+	path := (fmt.Sprintf("/orgs/%s/projects", org))
+
+	var projects []Project
+	err := c.readPaginatedResults(
+		path,
+		"application/vnd.github.inertia-preview+json",
+		func() interface{} {
+			return &[]Project{}
+		},
+		func(obj interface{}) {
+			projects = append(projects, *(obj.(*[]Project))...)
+		},
+	)
+	if err != nil {
+		return nil, err
+	}
+	return projects, nil
+}
+
+// GetProjectColumns returns the list of columns in a project.
+func (c *Client) GetProjectColumns(projectID int) ([]ProjectColumn, error) {
+	c.log("GetProjectColumns", projectID)
+	path := (fmt.Sprintf("/projects/%d/columns", projectID))
+
+	var projectColumns []ProjectColumn
+	err := c.readPaginatedResults(
+		path,
+		"application/vnd.github.inertia-preview+json",
+		func() interface{} {
+			return &[]ProjectColumn{}
+		},
+		func(obj interface{}) {
+			projectColumns = append(projectColumns, *(obj.(*[]ProjectColumn))...)
+		},
+	)
+	if err != nil {
+		return nil, err
+	}
+	return projectColumns, nil
+}
+
+// CreateProjectCard adds a project card to the specified project column.
+//
+// See https://developer.github.com/v3/projects/cards/#create-a-project-card
+func (c *Client) CreateProjectCard(columnID int, projectCard ProjectCard) (*ProjectCard, error) {
+	c.log("CreateProjectCard", columnID, projectCard)
+	if (projectCard.ContentType != "Issue") && (projectCard.ContentType != "PullRequest") {
+		return nil, errors.New("projectCard.ContentType must be either Issue or PullRequest")
+	}
+
+	// Not every create func has this block, TODO find out why and whether it should be here
```

Comment:
The `dry` should be on any call, but `fake` seems leaky -- @fejta how many places use the global fake versus making their own scoped ones?

Reply:
Not sure what you mean? I think the fake is mostly vestigial at this point?

Reply:
I've removed the reference to `fake` but left the reference to `dry`.